daft/dataframe/dataframe.py

```diff
@@ -1240,13 +1241,15 @@ def drop_nan(self, *cols: ColumnInputType):
 
     @DataframePublicAPI
     def drop_null(self, *cols: ColumnInputType):
-        """drops rows that contains NaNs or NULLs. If cols is None it will drop rows with any NULL value.
+        """Drops rows that contains NaNs or NULLs. If cols is None it will drop rows with any NULL value.
         If column names are supplied, it will drop only those rows that contains NULLs in one of these columns.
+
         Example:
             >>> df = daft.from_pydict({"a": [1.0, 2.2, 3.5, float("NaN")]})
             >>> df.drop_null()  # drops rows where any column contains Null/NaN values
             >>> df = daft.from_pydict({"a": [1.6, 2.5, None, float("NaN")]})
             >>> df.drop_null("a")  # drops rows where column a contains Null/NaN values
+
         Args:
```

Comment:
Made some small changes to the drop_null and drop_nan because they had some errors that were preventing the dataframe.py file from being ran in doctest.